### ISSUE TITLE
Add app links for signup activation link

### ISSUE THREAD

**Issue by harshitbansal05** [fossasia/susi_android#1089]
**Actual Behaviour**

On clicking the activation link sent to email account, the user is directed to the home screen of the website.

**Expected Behaviour**

Android app links must be used to redirect user to app, and then the request to the server must be made.

**Steps to reproduce it**

The feature needs an intent-filter for the LoginActivity.kt and can be implemented similar to:
https://developer.android.com/studio/write/app-link-indexing.html
Since, a retrofit request has to be made to the desired server, the nature of the request and response needs to be known.

**Would you like to work on the issue?**

Yes

**Comment by harshitbansal05** [comment]
@leto @rctay can you please review this?

**Comment by chiragw15** [comment]
@harshitbansal05 Yes, go ahead and add deep links. You can see API documentation in susi_server repo documentation.

**Comment by harshitbansal05** [comment]
@chiragw15 can you please share the github link where these requests are met "http://accounts.susi.ai/?token=<my_token>". I could'nt find it the susi server api docs.
Thanks.

**Comment by aniket-agarwal1999** [comment]
@harshitbansal05 @chiragw15 As the issue has been stagnant for quite some time, can I please work on this as I find it quite intriguing.

**Comment by batbrain7** [comment]
@codedsun will you take this up ?

**Comment by codedsun** [comment]
Yes

**Comment by codedsun** [comment]
@batbrain7 Close this issue, it has been fixed earlier.

**Comment by codedsun** [comment]
@batbrain7 I read about the api and get the server setup,  please can u help me like read about the how is the email sent to the client.
